app/HMS/Repositories/UserRepository.php

```diff
@@ -33,9 +34,12 @@ public function findOneByEmail(string $email);
     /**
      * @param  string $searchQuery
      * @param  bool $hasAccount limit to users with associated accounts
-     * @return array
+     * @param bool $paginate
+     * @param int    $perPage
+     * @param string $pageName
+     * @return User[]|array|\Illuminate\Pagination\LengthAwarePaginator
      */
-    public function searchLike(string $searchQuery, ?bool $hasAccount = false);
+    public function searchLike(string $searchQuery, ?bool $hasAccount = false, bool $paginate = false, $perPage = 15, $pageName = 'page');
```

Comment:
yeah the ? is declaring it a nullable type, php 7.1 introduced this, mean the value can be NULL or of the type
and yes into this case thats either Null, True or False